test/common/src/main/kotlin/org/codice/compliance/verification/profile/SingleSignOnProfileVerifier.kt

```diff
@@ -109,7 +109,7 @@ class SingleSignOnProfileVerifier(val response: Node) {
     private fun verifySingleLogoutIncludesSessionIndex(assertion: Node) {
         if (idpMetadata.descriptor == null) return
 
-        if (!idpMetadata.descriptor!!.singleLogoutServices.isNotEmpty()) return
+        if (idpMetadata.descriptor?.singleLogoutServices?.isNotEmpty() == true) return
```

Comment:
✏️ Don't need to test `boolean_value == true`. `boolean_value` is sufficient

Reply:
The result of `idpMetadata.descriptor?.singleLogoutServices?.isNotEmpty()` is a `Boolean?` which isn't a `Boolean`. After a quick google, testing equals true seems to be the simplest way.